elide-integration-tests/src/test/groovy/com/yahoo/elide/tests/FilterIT.groovy

```diff
@@ -1328,6 +1328,88 @@ class FilterIT extends AbstractIntegrationTestInitializer {
                         "Invalid query parameter: filter");
     }
 
+    @Test
+    public void testNull() {
```

Comment:
There were already tests here for the default filter dialect to test isNull and isNotNull.  You just need to add to those existing tests.